src/main/resources/templates/java/test/projectTemplate/pom.xml

```diff
@@ -71,6 +71,35 @@
                     <includeTests>false</includeTests>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.1</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>8.36</version>
+                    </dependency>
+                </dependencies>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
Is this now included in all projects?
I suggest we only include this if the option for static code analysis has been activated.
Otherwise builds will be slower and instructors might be confused by the many different entries in the pom.xml file

Reply:
There is a `%static-code-analysis-start%` before this on line 64 which is not rendered in the diff by default.

Reply:
@MaisiKoleni is right. The new dependencies are only included if static code analysis is enabled.